sdks/python/apache_beam/examples/inference/online_clustering/clustering_pipeline/main.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import sys
+
+import apache_beam as beam
+from apache_beam.io.gcp.pubsub import ReadFromPubSub
+from apache_beam.ml.inference.base import KeyedModelHandler, RunInference
+from apache_beam.ml.inference.pytorch_inference import PytorchModelHandlerKeyedTensor
+from transformers import AutoConfig
+
+import config as cfg
+from pipeline.options import get_pipeline_options
+from pipeline.transformations import (
+    Decode,
+    GetUpdates,
+    ModelWrapper,
+    NormalizeEmbedding,
+    StatefulOnlineClustering,
+    tokenize_sentence,
+)
+
+
+def parse_arguments(argv):
+    parser = argparse.ArgumentParser(description="online-clustering")
+
+    parser.add_argument(
+        "-m",
+        "--mode",
+        help="Mode to run pipeline in.",
+        choices=["local", "cloud"],
+        default="local",
+    )
+    parser.add_argument(
+        "-p",
+        "--project",
+        help="GCP project to run pipeline on.",
+        default=cfg.PROJECT_ID,
+    )
+
+    args, _ = parser.parse_known_args(args=argv)
+    return args
+
+
+class PytorchNoBatchModelHandler(PytorchModelHandlerKeyedTensor):
```

Comment:
Could we add a comment that links to https://github.com/apache/beam/issues/21863 and mentions that we can remove this workaround once it is fixed?